user/api.go

```diff
@@ -274,6 +279,11 @@ func (a *Api) CreateCustodialUser(res http.ResponseWriter, req *http.Request, va
 		} else {
 			a.logMetricForUser(newCustodialUser.Id, "custodialusercreated", sessionToken, map[string]string{"server": strconv.FormatBool(tokenData.IsServer)})
 			a.sendUserWithStatus(res, newCustodialUser, http.StatusCreated, tokenData.IsServer)
+			if err := a.userEventsNotifier.NotifyUserCreated(req.Context(), *newCustodialUser); err != nil {
```

Comment:
`a.sendUserWithStatus(res, newCustodialUser, http.StatusCreated, tokenData.IsServer)` must be placed after this block

Reply:
I suggest removing this event completely if it's not used by marketo.

Reply:
removing